### DISCUSSION TITLE
occ taking 30 seconds to react

### DISCUSSION THREAD

**Discussion by fmarcabru** [nextcloud/all-in-one#3031] [Questions] [answered]
hi,
i have a new installation of AIO in reverse proxy mode.  server details are as below
Operating System: Linux 5.15.108-1-pve x86_64
CPU: Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz (6 cores)
Memory: 15.47 GB
Server time: Tue Jul 25 21:47:05 AEST 2023
Uptime: 2 days, 6 hours, 29 minutes, 1 seconds

I am seeing a 30 second delay in occ, bare ./occ
real    0m30.325s
user    0m0.284s
sys     0m0.033s

compared to a nextcloud container on the same box 
real    0m0.273s
user    0m0.212s
sys     0m0.050s

I see the following messages in the web logs
GuzzleHttp\Exception\ConnectException: cURL error 28: Connection timed out after 30000 milliseconds (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://nextcloud.aaa.bbb/standalone-signaling/api/v1/room/s2x9dex4

not sure if it is related but both have 30secs as common denominator

any ideas on what can be happening or what to check?

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Reply by fmarcabru** [reply]
great, thanks. I disabled talk and it now  runs in 270ms
